outputs/subgraph_urls.json

```diff
@@ -75,13 +74,13 @@
     }
   },
   "mode": {
-    "core": "https://api.studio.thegraph.com/query/75376/balancer-mode-v2/version/latest",
+    "core": "https://api.studio.thegraph.com/proxy/75376/balancer-mode-v2/version/latest",
```

Comment:
What is this change about?

Reply:
the changes to `subgraph_urls.json` is just the result of changes in the various config files (frontend/sdk). in this case in https://raw.githubusercontent.com/balancer/frontend-v2/develop/src/lib/config/mode/index.ts